platforms/documentation/docs/src/snippets/bestPractices/avoidInternal/tests/avoidInternal.sample.conf

```diff
@@ -0,0 +1,9 @@
+commands: [{
+    executable: gradle
+    args: badTask
+    expected-output-file: avoidInternal.bad.out
```

Comment:
🤔 The expected output files are optional.  If there is no need to validate output in a snippet, you can remove the argument from here and delete the files.  Sometimes it's enough just to ensure that an example build runs successfully.